nimsuggest/tests/top_highlight.nim

```diff
@@ -0,0 +1,11 @@
+import json
+
+%*{}#[!]#
+
+discard """
+$nimsuggest --tester $file
+>highlight $1
+highlight;;skModule;;1;;7;;4
+highlight;;skMacro;;3;;0;;2
+highlight;;skMacro;;3;;0;;2
```

Comment:
Huh? So it produces this twice? Isn't that a bug on its own?

Reply:
Yea, see https://github.com/nim-lang/nimsuggest/issues/77

Some symbols got passed through `suggestSym` more than once (due to the amount of hooks in the compiler), hence the duplication.